### PR TITLE
W-13588449: Parse template target value breaking change.

### PR DESCRIPTION
**Pull request by IvanAndresFritzler**

### CODE REVIEW

**Comment by IvanAndresFritzler** [comment]
--validate

**Review by elrodro83 — DISMISSED** [review]

File: modules/core-components/src/test/java/org/mule/runtime/core/internal/processor/simple/ParseTemplateProcessorTestCase.java [hunk 1]
```diff
@@ -67,6 +78,19 @@ public class ParseTemplateProcessorTestCase extends AbstractMuleTestCase {
...
+  public SystemProperty isUseLegacyDefaultTargetValue;
+
+  @Parameters(name = "isUseLegacyTargetValueDefaultValue: {0}")
+  public static Collection<String> getParameters() {
+    return Arrays.asList("true", "false");
+  }
+
+  public ParseTemplateProcessorTestCase(String isUseLegacyDefaultTargetValue) {
+    this.isUseLegacyDefaultTargetValue =
+        new SystemProperty(MuleSystemProperties.PARSE_TEMPLATE_USE_LEGACY_DEFAULT_TARGET_VALUE, isUseLegacyDefaultTargetValue);
```
Comment (fsgonz) [inline modules/core-components/src/test/java/org/mule/runtime/core/internal/processor/simple/ParseTemplateProcessorTestCase.java:91]: static import.

**Review by fsgonz — CHANGES_REQUESTED** [review]

**Review by fsgonz — APPROVED** [review]

**Comment by IvanAndresFritzler** [comment]
--validate

**Comment by IvanAndresFritzler** [comment]
--validate

**Pull request merged**